docs/site/Behind-the-scene.md

```diff
@@ -1,22 +1,75 @@
 ---
 lang: en
-title: 'Components'
-keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI
+title: 'Behind the scene'
```

Comment:
A thought: I see frameworks like [angular](https://angular.io/docs) uses word "Techniques" , I think it describes the same concept and concise as well.

Reply:
I don't think Angular is a good source of inspiration. When I look at their docs for [Animation technique](https://angular.io/guide/transition-and-triggers), I see a mix of tutorials, explanations and how-to guides. In my PR, I am trying to move away from that style.